src/Commands/CreateDraftsMigration.php

```diff
@@ -45,8 +43,10 @@ public function handle()
 
     public function getTables()
     {
-        $tables = DB::select('SHOW TABLES');
-        return array_map('current', $tables);
+        $schema = collect(DB::getDoctrineSchemaManager()->listTableNames())->map(function (string $item, int $key) {
```

Comment:
`return collect(DB::getDoctrineSchemaManager()->listTableNames())->all()`

Could we have it be just that? Having the map function seems quite pointless.

Reply:
agree, change implemented